python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -361,6 +361,9 @@ def __reduce__(self):
             self._reader_ref,
         )
 
+    def __repr__(self) -> str:
```

Comment:
Here as well?

Reply:
Yeah. I also found that I forgot to update it last night. We can update it in any new ADAG PRs so that we don't need to run the CI again. This PR currently blocks the master branch.